ex/high_throughput_inference/standalone_worker_manager.py

```diff
@@ -124,13 +119,15 @@ def service_as_dragon_proc(
         "--batch_size",
         type=int,
         default=1,
-        help="How many requests the workers will try to aggregate before processing them",
+        help="How many requests the workers will try "
```

Comment:
nitpick... I'm not a fan of "how many requests..." for some reason. Consider

```suggestion
        help="Maximum number of samples to aggregat into a batch inference request"
```

Reply:
Done